client/components/token-field/test/index.jsx

```diff
@@ -32,89 +35,81 @@ const charCodes = {
 };
 
 describe( 'TokenField', function() {
-	var reactContainer, wrapper, tokenFieldNode, textInputNode, TokenFieldWrapper;
+	let wrapper, tokenFieldNode, textInputNode, TokenFieldWrapper, mount;
 
 	function setText( text ) {
-		TestUtils.Simulate.change( textInputNode, { target: { value: text } } );
+		textInputNode.simulate( 'change', { target: { value: text } } );
 	}
 
 	function sendKeyDown( keyCode, shiftKey ) {
-		TestUtils.Simulate.keyDown( textInputNode, {
+		tokenFieldNode.simulate( 'keyDown', {
 			keyCode: keyCode,
 			shiftKey: !! shiftKey
 		} );
 	}
 
 	function sendKeyPress( charCode ) {
-		TestUtils.Simulate.keyPress( textInputNode, {
+		tokenFieldNode.simulate( 'keyPress', {
 			charCode: charCode
 		} );
 	}
 
+	function getNodeInnerHtml( node ) {
+		const div = document.createElement( 'div' );
+		div.innerHTML = node.html();
+		return div.firstChild.innerHTML;
+	}
+
 	function getTokensHTML() {
-		var textNodes = tokenFieldNode.querySelectorAll( '.token-field__token-text' );
-		return Array.prototype.slice.call( textNodes ).map( function( el ) {
-			return el.innerHTML;
-		} );
+		const textNodes = tokenFieldNode.find( '.token-field__token-text' );
+
+		return textNodes.map( getNodeInnerHtml );
 	}
 
 	function getSuggestionsHTML( selector ) {
-		var suggestionNodes = tokenFieldNode.querySelectorAll( selector || '.token-field__suggestion' );
-		return Array.prototype.slice.call( suggestionNodes ).map( getSuggestionNodeHTML );
+		const suggestionNodes = tokenFieldNode.find( selector || '.token-field__suggestion' );
+
+		return suggestionNodes.map( getSuggestionNodeHTML );
 	}
 
 	function getSuggestionNodeHTML( node ) {
-		var suggestionContent = node.childNodes[0];
-		if ( suggestionContent.nodeName === '#text' ) {
-			// This suggestion is not part of a partial match; just return
-			// the whole suggestion
-			return node.innerHTML;
+		if ( ! node.find( 'span' ).length ) {
```

Comment:
Will this break when we upgrade to React 15.0 ? (Which I'd like to explore in the very near future)

Related: https://facebook.github.io/react/blog/2016/04/07/react-v15.html#no-more-extra-ltspangts